doc/snippets/Utility.cpp

```diff
@@ -605,6 +606,25 @@ struct {
 } position;
 };
 }
+
+{
+/* [Sha1-usage] */
+Utility::Sha1 sha1;
+
+/* Add data without delimiter */
+sha1 << std::string("corrade");
+
+/* Add data with delimiter */
+sha1 << Containers::arrayView("edarroc", 8);
+
+/* Compute digest as a hex string */
+Utility::Sha1::Digest d = sha1.digest();
+std::string hexDigest = d.hexString();
+
+/* Compute digest of string without delimiter */
+Utility::Sha1::digest("corrade");
+/* [Sha1-usage] */
```

Comment:
Err ... no, too confusing 😅 The below is not ideal either tho, I'd need to think more for a better example:

```cpp
Utility::Sha1 sha1;

/* Add 7 bytes of string data */
sha1 << std::string{"corrade"};

/* Add four bytes of binary data */
const char data[] = { '\x35', '\xf6', '\x00', '\xab' };
sha1 << Containers::arrayView(data);

/* Print the digest as a hex string */
Utility::Debug{} << sha1.digest().hexString();

/* Shorthand variant */
Utility::Debug{} << Utility::Sha1::digest("corrade");
```

Reply:
I changed the snippet to this for now (+ DebugStl.h include).